src/utils/v2v3/loadV2V3Contract.ts

```diff
@@ -105,14 +114,27 @@ const loadJuiceboxV2Contract = async (
   contractName: V2V3ContractName,
   network: NetworkName,
 ) => {
-  const contractOverride = CONTRACT_ABI_OVERRIDES[contractName]
+  const contractOverride = V2_CONTRACT_ABI_OVERRIDES[contractName]
   const version = contractOverride?.version ?? 'latest'
   const filename = contractOverride?.filename ?? contractName
   return await import(
     `@jbx-protocol/contracts-v2-${version}/deployments/${network}/${filename}.json`
   )
 }
 
+const loadJuiceboxV3Contract = async (
+  contractName: V2V3ContractName,
+  network: NetworkName,
+) => {
+  try {
+    return await import(
+      `@jbx-protocol/juice-contracts-v3/deployments/${network}/${contractName}.json`
+    )
+  } catch (_) {
+    return undefined
```

Comment:
this will be undefined because we're iterating over the V2V3ContractName, which contains the deprecated contracts, only applicable to V2.

We should come up with a better solution for this long-term, but this'll work for now.